website/src/controllers/api-v2/groups.js

```diff
@@ -693,6 +693,10 @@ api.invite = function(req, res, next){
   if (group.privacy === 'private' && !_.contains(group.members,res.locals.user._id)) {
     return res.json(401, {err: "Only a member can invite new members!"});
   }
+  if (group.type === 'party' && group.isOverPartyCap) {
+    return res.json(400, {err: "Over 30 member party cap"});
```

Comment:
Yeah, I've actually already done this.  I have a commit ready that I haven't pushed.